src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+    String getPrintableString();
```

Comment:
Header comments missing. `Write descriptive header comments for all public classes/methods`